### ISSUE TITLE
center_net _custom 

### ISSUE THREAD

**Issue by austinmw** [dmlc/gluon-cv#1454]
Hi, I noticed that the center_net detection model does not have a _custom version as the other detection models do. Is this disadvantageous for using this base or should the coco/voc versions be fine?

**Comment by zhreshold** [comment]
Yes, the custom versions of centernets are not available yet, as we are tight on the new features, anyone who can contribute the functionality is super welcome.
